README.md

```diff
@@ -133,17 +133,17 @@ Results using Ruby 3.1.1:
 
 Results using Ruby 2.7.5 (because these gems raise errors in Ruby 3.x):
 
-|Method arguments|`DDMemoize` (1.0.0)|`Memoist` (0.16.2)|`Memoized` (1.0.2)|`Memoizer` (1.0.3)|
```

Comment:
Definitely easier to review using GitHub's rich diff: https://github.com/panorama-ed/memo_wise/pull/275/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5